src/OpenTK/MouseCursor.cs

```diff
@@ -28,28 +28,55 @@
 #endregion
 
 using System;
+using System.Collections;
+using System.Collections.Generic;
 
 namespace OpenTK
 {
     /// <summary>
+    /// Stores information about frame timing
+    /// </summary>
+    public struct MouseCursorFrameInformation
```

Comment:
Actually we'd need a new name as I renamed the original MouseCursor to MouseCursorFrame (=> Image without timing information) while MouseCursorInformation just associates a timing information with an actual frame.
Embedding timing info within the actual MouseCursorFrame would do no good either, as this would prevent one using the same frame with different durations within an animation sequence. (One would require to duplicate the actual image in this case -> Bad idea)

Maybe something like: MouseCursorTimedFrame